README.md

```diff
@@ -301,6 +301,45 @@ import exampleGo from '!!raw-loader!./code-example.go'
 ```
 ````
 
+### Mermaid graphs
```

Comment:
If I don't know what Mermaid is I won't know you're talking about diagrams.

Reply:
is "graphs" not clear enough? 
Could be 
Mermaid graphs and diagrams

Reply:
We don't really do graphs, do we?

<img width="478" alt="image" src="https://user-images.githubusercontent.com/29116492/205313616-21b9bf30-764e-46d9-9628-72ab46558c6c.png">


> [About Mermaid](https://mermaid-js.github.io/mermaid/#/?id=about-mermaid)
Mermaid lets you create diagrams and visualizations using text and code.

Reply:
gotcha